encord/orm/dataset.py

```diff
@@ -766,8 +766,11 @@ def get_str(self) -> str:
             return "AZURE_STR"
         elif self == StorageLocation.OTC:
             return "OTC_STR"
+        else:
+            return "NEW_STORAGE"
```

Comment:
This is no no, this should be `raise NotImplementedError("...")`

Reply:
Updated, thanks!